wiki/Music_theory/Measure/en.md

```diff
@@ -0,0 +1,15 @@
+---
+stub: true
+tags:
+  - bar
+---
+
+# Measure
+
+A **measure** (or **bar**) in music theory refers to a single unit of time featuring a specific number of [beats](/wiki/Music_theory/Beat) played at a particular [tempo](/wiki/Music_theory/Tempo).
```

Comment:
I have a feeling that this page may be safely combined with the one about time signatures -- what do you think? like, is there any more material outside of the article that could be brought in to remove `stub: true`?

Reply:
probably, i wouldn't know how to properly combine them though

Reply:
to me this stub doesn't look too bad as is